### PR TITLE
add new endpoints and fix default dates in postman request

### PR DESCRIPTION
**Pull request by phoenixy1**
- Adds transfers sweep endpoints
- Adds examples for a couple of endpoints where they were missing, like /liabilities/get
- Updates default date range for investments transactions request

### CODE REVIEW

File: Plaid API Endpoints Collection.json [hunk 1]
```diff
@@ -4235,7 +4235,7 @@
 			"name": "Liabilities",
 			"item": [
 				{
-					"name": "OK",
+					"name": "Retrieve Liabilities",
```
Comment (ToddKerpelman) [inline Plaid API Endpoints Collection.json:4238]: I noticed sometimes we use "OK" and sometimes we use the endpoint name. Are we gradually renaming these across the json file?

File: Plaid API Endpoints Collection.json [hunk 2]
```diff
@@ -6388,6 +6489,203 @@
 							"body": "{\n \"request_id\": \"mdqfuVxeoza6mhu\"\n}"
 						}
 					]
+				},
+				{
+					"name": "Get a sweep",
```
Comment (ToddKerpelman) [inline Plaid API Endpoints Collection.json:6494]: I was unable to test these in postman (even in Sandbox) but they look okay in the Postman app.

**Review by ToddKerpelman — APPROVED** [review]
LGTM

File: Plaid API Endpoints Collection.json (hunk 1 above)
Comment (phoenixy1) [reply Plaid API Endpoints Collection.json:4238]: They are supposed to read "OK". I'll fix that.

**Pull request merged**